src/components/pager/pager.component.js

```diff
@@ -165,7 +156,7 @@ const Pager = ({
   const renderTotalRecords = () =>
     showTotalRecords && (
       <>
-        {totalRecords} {records(totalRecords)}
+        {totalRecords} {l.pager.records(totalRecords)}
```

Comment:
`records` should return the full string instead of the component concatenating the number.
```suggestion
        {l.pager.records(totalRecords)}
```

Reply:
changed, thanks